examples/Workflow/README.md

```diff
@@ -11,105 +11,129 @@ This Dapr workflow example shows how to create a Dapr workflow (`Workflow`) and
 
-To run the workflow web app locally, run this command in the `WorkflowWebApp` directory:
+To run the workflow web app locally, two separate terminal windows are required.
+In the first terminal window, down the `WorkflowConsoleApp` directory, run the following command to start the program itself:
 
 ```sh
-dapr run --app-id wfwebapp dotnet run
+dotnet run
 ```
 
-The application will listen for HTTP requests at `http://localhost:10080`.
+Next, in a separate terminal window, start the dapr sidecar:
 
-To start a workflow, use the following command to send an HTTP POST request, which triggers an HTTP API that starts the workflow using the Dapr Workflow client. Two identical `curl` commands are shown, one for Linux/macOS (bash) and the other for Windows (PowerShell). The body of the request is used as the input of the workflow.
+```sh
+dapr run --app-id wfwebapp --dapr-grpc-port 4001 --dapr-http-port 3500
+```
+
+Dapr will listen for HTTP requests at `http://localhost:3500`.
+
+This workflow example utilizes a redis statestore to simulate the purchasing of items and restocking of inventory. The console prompts will provide directions on how to both purchase and restock items.
 
-On Linux/macOS (bash):
+To start a workflow, you have two options:
+Option A: Follow the directions from the console prompts.
+
+Option B: Use the workflows API and send a request to Dapr directly. Examples are included below as well as in the "demo.http" file down the "WorkflowConsoleApp" directory.
+
+Two identical `curl` commands are shown, one for Linux/macOS (bash) and the other for Windows (PowerShell). The body of the request is used as the input of the workflow. 
+
+Make note of the "xxx" in the commands below. This represents the unique identifier for the workflow run and can be replaced with any identifier of your choosing.
 
 ```bash
-curl -i -X POST http://localhost:10080/orders \
+curl -i -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/xxx/start \
   -H "Content-Type: application/json" \
-  -d '{"name": "Paperclips", "totalCost": 99.95, "quantity": 1}'
+  -d '{ "input" : {"Name": "Paperclips", "TotalCost": 99.95, "Quantity": 1}}'
 ```
 
 On Windows (PowerShell):
 
 ```powershell
-curl -i -X POST http://localhost:10080/orders `
+curl -i -X POST http://localhost:3500/v1.0-alpha1/workflows/dapr/OrderProcessingWorkflow/xxx/start `
   -H "Content-Type: application/json" `
-  -d '{"name": "Paperclips", "totalCost": 99.95, "quantity": 1}'
+  -d '{ "input" : {"Name": "Paperclips", "TotalCost": 99.95, "Quantity": 1}}'
 ```
 
-If successful, you should see a response like the following, which contains a `Location` header pointing to a status endpoint for the workflow that was created with a randomly generated 8-digit ID:
+If successful, you should see a response like the following, which contains a `Location` header pointing to a status endpoint for the workflow that was created with the identifier that you provided. 
```

Comment:
I guess it's not returning any `Location` header, what I'm missing?
<img width="590" alt="image" src="https://user-images.githubusercontent.com/5839364/217020670-78f8d472-3b02-4c3e-90b1-42517729ac7f.png">

Reply:
I’m planning to open a bug to track the missing Location header from the StartWorkflow API. Previously, we were returning it from the custom web API. For now, we should remove this from the directions.

Reply:
I removed the `Location` header from the response example and removed mention of it from the text. I also went ahead and removed the `Traceparent` headers to avoid distracting the user from the primary objective of this example.